kroxylicious-filters/kroxylicious-encryption/src/test/java/io/kroxylicious/filter/encryption/EnvelopeEncryptionFilterTest.java

```diff
@@ -186,7 +186,7 @@ void shouldDecryptEncryptedRecords() {
 
         when(keyManager.decrypt(any(), anyInt(), assertArg(records -> assertThat(records).hasSize(1)), any(Receiver.class))).thenAnswer(invocationOnMock -> {
             final Receiver receiver = invocationOnMock.getArgument(3);
-            receiver.accept(new TestingRecord(plainTextPayload.messageBytes()), plainTextPayload.messageBytes(), Record.EMPTY_HEADERS);
+            receiver.accept(RecordTestUtils.record(plainTextPayload.messageBytes()), plainTextPayload.messageBytes(), Record.EMPTY_HEADERS);
```

Comment:
I could imagine the no-header case being pretty common for testing, maybe we should have a convenience like `RecordTestUtils.record(keyBytes, valueBytes)` .

Reply:
This `Record.EMPTY_HEADERS` is actually being passed as an argument to `Receiver`, not `RecordTestUtils.record()`